pkg/iac-providers/terraform/v12/resource.go

```diff
@@ -49,15 +55,40 @@ func CreateResourceConfig(managedResource *hclConfigs.Resource) (resourceConfig
 
 	// create a resource config
 	resourceConfig = output.ResourceConfig{
-		ID:     fmt.Sprintf("%s.%s", managedResource.Type, managedResource.Name),
-		Name:   managedResource.Name,
-		Type:   managedResource.Type,
-		Source: managedResource.DeclRange.Filename,
-		Line:   managedResource.DeclRange.Start.Line,
-		Config: goOut,
+		ID:        fmt.Sprintf("%s.%s", managedResource.Type, managedResource.Name),
+		Name:      managedResource.Name,
+		Type:      managedResource.Type,
+		Source:    managedResource.DeclRange.Filename,
+		Line:      managedResource.DeclRange.Start.Line,
+		Config:    goOut,
+		SkipRules: getSkipRules(c.rangeSource(hclBody.Range())),
 	}
 
 	// successful
 	zap.S().Debugf("created resource config for resource '%s', file: '%s'", resourceConfig.Name, resourceConfig.Source)
 	return resourceConfig, nil
 }
+
+// getSkipRules returns a list of rules to be skipped. The rules to be skipped
+// can be set in terraform resource config with the following comma separated pattern:
+// #ts:skip=AWS.S3Bucket.DS.High.1043, AWS.S3Bucket.DS.High.1044
+func getSkipRules(body string) []string {
```

Comment:
This could be useful for other IaC... Maybe place in utils for now?

Reply:
Done!